android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/RibActivity.kt

```diff
@@ -177,13 +176,15 @@ abstract class RibActivity : CoreAppCompatActivity(), ActivityStarter, Lifecycle
   }
 
   override fun onUserLeaveHint() {
-    lifecycleRelay.accept(create(ActivityLifecycleEvent.Type.USER_LEAVING))
+    lifecycleFlow.tryEmit(create(ActivityLifecycleEvent.Type.USER_LEAVING))
     super.onUserLeaveHint()
   }
 
   override fun onWindowFocusChanged(hasFocus: Boolean) {
     super.onWindowFocusChanged(hasFocus)
-    callbacksRelay.accept(createWindowFocusEvent(hasFocus))
+
+    val result = callbacksFlow.tryEmit(createWindowFocusEvent(hasFocus))
+    val s = ""
```

Comment:
I don't get it. Left over from local testing?

Reply:
junk, already removed.